features/events/traps/src/main/java/org/opennms/netmgt/trapd/TrapReceiverSnmp4jImpl.java

```diff
@@ -68,8 +64,6 @@
 public class TrapReceiverSnmp4jImpl implements TrapReceiver, TrapNotificationListener,TrapProcessorFactory {
```

Comment:
I was not able to remove the run method completely because TrapReceiver class is implementing Runnable interface. so there is a run method with empty body is still present.